concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -32,6 +33,44 @@ def parse_command_line(command_line):
     return cluster_url
 
 
+def upload_new_file_on_cluster(task_id = '0', part_id = '0', current_time = 0):
+
+    os.system('touch result')
+    os.system('echo "{}" > result'.format(task_id))
+    file_size       = int(os.popen("wc -c < result").readlines()[0])
```

Comment:
```
import os
os.path.getsize(file_name)
```

would be more pythonic.

Reply:
Yeah. Hundred times this. Please no `popen()`, `system()` or any other calls that spawn a separate process unless your fully aware of all the consequences of this. We should probably add this to coding style as a red flag that something is fishy.

As a general rule, if you're embedding bash commands in your python code, 99% of the time you're doing something wrong.